### PR TITLE
DEV: added MCH_Staff model, poetry.lock should contain psycopg2, chaned settings.py to connect to postgres database on localhost

### PR DESCRIPTION
**Pull request by AndreaHabib**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,25 +45,45 @@ pre-commit install
 pre-commit run --all-files
 
 echo "setup is finished 🥳"
+
+#to use psycopg2
+poetry add psycopg2
```
[the remark this answers is not in the document]
Comment (AndreaHabib) [reply README.md:50]: oh yeah right my bad :+1:

**Comment by dhyxy** [comment]
@AndreaHabib I just added a few snippets that might make the setup a bit easier, let me know what you think!

**Comment by AndreaHabib** [comment]
That looks way better now, i was actually gonna ask u how to use settings.ini as env var 😅, but yeah nice stuff

**Comment by dhyxy** [comment]
@AndreaHabib was trying to switch u over to be a reviewer 😅 let me know how this looks, I just renamed it to follow [this](https://stackoverflow.com/questions/385895/django-model-naming-convention). it shows up like this in the admin page now 
image

**Comment by AndreaHabib** [comment]
I have no idea what ur doing with the migrations, but yeah i will take a look at them tomorrow

**Review by dhyxy — CHANGES_REQUESTED** [review]

File: caresomalia/api/models.py [hunk 2]
```diff
@@ -31,3 +31,64 @@ class Meta:
...
+
+    class Meta:
+        verbose_name = "Community Health Worker"
+        verbose_name_plural = "Community Health Workers"
+
+    def __str__(self):
+        return f"CH Worker {self.worker_name}, {self.email}"
+
+
+class AssignPatient(models.Model):
```
Comment (dhyxy) [inline caresomalia/api/models.py:69]: @AndreaHabib do you think we should rename this to `PatientAssignment` by any chance or nah?

**Review by dhyxy — APPROVED** [review]

**Pull request merged**